src/transduce.js

```diff
@@ -0,0 +1,6 @@
+var _reduce = require('./internal/_reduce');
+var curryN = require('./curryN');
+
+module.exports = curryN(4, function(xf, fn, acc, ls) {
```

Comment:
Is `ls` short for "list"? We don't use this name elsewhere. Could we use `list` instead?